README.md

```diff
@@ -136,3 +136,28 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
```

Comment:
```suggestion
## Creating release tags

```

Reply:
Is this just "new line following a heading" for Markdown? I guess I need to get my MD linting setup to catch this. Sorry.

Reply:
Yeah, just a new line after the heading. No worries.